scripts/machine-launch-script.sh

```diff
@@ -308,6 +308,7 @@ set -o pipefail
         boto3-stubs==1.21.6 \
         botocore-stubs==1.24.7 \
         mypy-boto3-s3==1.21.0 \
+        yamale==4.0.4 \
```

Comment:
This might be more a @timsnyder-siv question, but is it wise to pin these down? It seems to me that providing an inequality, or major version constraint, might make the environment more solvable?

Reply:
Agreed. However, I think we like pinning things so that we don't get any weird issues where a version changes and a bug occurs silently.